doc/chapter-password.txt

```diff
@@ -0,0 +1,31 @@
+When using external APIs for newsfeeds, the user has different options for how
+to provide the password to authenticating against the API.
+Instead of directly writing the password into the configuration file,
+the user can choose to provide a file containing the password or an external
+command that evaluates to the user's password.
+This is useful if the user wants to share the configuration file online or
+store the password in a password manager or the systems keyring.
+
+For example when using the API for "The Old Reader",
+Newsbeuter will first check the `oldreader-password` configuration variable,
+if this variable is set (meaning it's present and not empty),
+then its value will be used for authentication.
+If `oldreader-password` is not set (meaning it's not present or empty)
```

Comment:
You are right. Another case the ambiguity of natural languages. Didn't even recognize it could be read that way